.gitignore

```diff
@@ -7,3 +7,4 @@ Config/secrets/
 .DS_Store
 .swift-version
 CMakeLists.txt
+.env
```

Comment:
Maybe we should make this `*.env` since you could have different env files for different setups.

Reply:
Good point. Also the implementation supports .env files with alternative names. I'll update it.